compiler/compile/OMRCompilation.hpp

```diff
@@ -353,6 +353,7 @@ class OMR_EXTENSIBLE Compilation
    void verifySymbolHasBeenValidated(void *symbol) { return; }
    void incrementHeuristicRegion() { return; }
    void decrementHeuristicRegion() { return; }
+   bool validateImplementer(TR_ResolvedMethod *implementer) { return true; }
```

Comment:
Maybe comment that this facility isn't used directly in OMR, but can be implemented by consuming projects, along with a description of how the return value will be interpreted by OMR?

Reply:
I can't help asking "a valid implementer of what?" If the question is really inlining specific, could the name instead be `validateTargetToBeInlined()` or something a little more meaningful?